### ISSUE TITLE
Enhancement: Switch to "poetry"

### ISSUE THREAD

**Issue by blos** [user1342/Tweezer#5]
Currently there are two ways of installing the dependencies for this project.
The first one is using pip install -r requirements.txt.
This installs the modules that Tweezer depends on, however does not install Tweezer itself as usable package.
The consequence of this is that using import Tweezer will not work.
Using pip install . (being in the root directory of the repository) will install Tweezer properly, such as import Tweezer will work.
For this matter the setup.py file is utilized.
Note that the latter variation will execute python code for the installing the package.
For reference this was issued in [PEP 518](https://peps.python.org/pep-0518/).

[Poetry](https://python-poetry.org/) is a tool for packaging and dependency management.
In short, poetry pinpoints dependency versions which can help isolate breaking upgrades (in the future) and further building (and potentially publishing) the package is as easy as using poetry build or poetry publish, which is quite easy and handy way of doing this steps.

Relates to #3 (Enhancement: Multiple Ideas for this project)
